src/applications/facility-locator/constants/index.js

```diff
@@ -56,3 +56,8 @@ export const LOCATION_OPTIONS = [
  * Defines the ± change in bounding box size for the map when changing zoom
  */
 export const BOUNDING_RADIUS = 0.75;
+
+/**
+ *Defines the marker letter list
+ */
+export const letters = [...'abcdefghijklmnopqrstuvwxyz'.toUpperCase()];
```

Comment:
Oh cool, I didn't know you could use the spread operator on a string!

I think renaming to something like `MARKER_LETTERS` (or `ALPHABET` is kind of a cool name) and as a `Set` could be really cool. Like -

```
const MARKER_LETTERS = new Set([...'abcdefghijklmnopqrstuvwxyz'.toUpperCase()]);
```

Then, instead of looking up by index, you could use the iterator function like -

```
const marketLetters = MARKER_LETTERS.values();

// ...
markerText: marketLetters.next()
```